### PR TITLE
[refactor] Upgrade to MySQL 5.7.37 to reduce unnecessary CVE issues

### PR DESCRIPTION
**Pull request by aopangzi**
## Problem Summary:

Upgrade to MySQL 5.7.37 to reduce unnecessary CVE issues

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (No)
3. Has document been added or modified: (No)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by morningman** [comment]
Hi @aopangzi , thanks for your contribution.
And please also modify the document: 
```
docs/zh-CN/administrator-guide/variables.md
docs/en/administrator-guide/variables.md
```
to add new variables

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 1]
```diff
@@ -50,6 +50,9 @@
     public static final String RESOURCE_VARIABLE = "resource_group";
     public static final String AUTO_COMMIT = "autocommit";
     public static final String TX_ISOLATION = "tx_isolation";
+    public static final String TX_READ_ONLY = "tx_read_only";
+	public static final String TRANSACTION_READ_ONLY = "transaction_read_only";
```
[the remark this answers is not in the document]
Comment (aopangzi) [reply fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java]: Thank you for your advice. It has been corrected

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 2]
```diff
@@ -973,6 +1000,9 @@ private void readFromStream(DataInput in) throws IOException {
         charsetConnection = Text.readString(in);
         charsetClient = Text.readString(in);
         txIsolation = Text.readString(in);
+	    transactionIsolation = Text.readString(in);
```
[the remark this answers is not in the document]
Comment (aopangzi) [reply fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java]: Yes, you're right. The tx_isolation and tx_read_only variables are now(5.7.20) deprecated and will be removed in MySQL 8.0 .So add transaction_ isolation。

File: fe/fe-core/src/main/java/org/apache/doris/qe/GlobalVariable.java [hunk 3]
```diff
@@ -47,7 +47,7 @@
             Version.DORIS_BUILD_VERSION + "-" + Version.DORIS_BUILD_SHORT_HASH;
 
     @VariableMgr.VarAttr(name = VERSION, flag = VariableMgr.READ_ONLY)
-    public static String version = "5.1.0";
+    public static String version = "5.7.37";
```
[the remark this answers is not in the document]
Comment (aopangzi) [reply fe/fe-core/src/main/java/org/apache/doris/qe/GlobalVariable.java:50]: Thank you for your advice. I'll study it

**Review by hf200012 — DISMISSED** [review]
LGTM

**Comment by morningman** [comment]
Hi @aopangzi , please rebase the master to solve the conflict.

**Review by hf200012 — APPROVED** [review]
LGTM

**Pull request merged**